src/frontend/js/types/Joanie.ts

```diff
@@ -228,6 +235,10 @@ export interface ApiResourceInterface<
   delete?: (id: TData['id']) => Promise<void>;
 }
 
+export interface UserWishlistCoursesFilters extends ResourcesQuery {
+  courseCode?: string;
+}
+
```

Comment:
What do you think to move this interface into `hooks/useUserWishlistCourses` ? This is what I did for `useOrders` hook.